### DISCUSSION TITLE
Regarding Control Volume

### DISCUSSION THREAD

**Discussion by tolstoydebbarma** [xcompact3d/Incompact3d#252] [Q&A] [answered]
Hi all, I want to simulate a sinusoidal cylinder motion translating in x-direction. How can I change the bounds of the control volume so that it remains near the moving cylinder?

**Comment by slaizet** [comment, marked as answer]
Hi,
You would just need to have the control volume big enough so that the cylinder is always in the control volume. The size of the control volume does not affect the calculation of lift/drag.
Best,
Sylvain
